nchoosek/solver/common.py

```diff
@@ -58,7 +59,54 @@ def __init__(self):
         self.solutions = None
         self.tallies = None
         self.energies = None
-        self.jobs = None
         self.jobIDs = None
         self.qubits = None
         self.depth = None
+        self.times = None
+        self.quantum_instance = None
+
+    def __repr__(self):
+        ret = ""
+        if self.solutions:
+           ret += "top solution: " + str(self.solutions[0]) + "\n"
+           ret += "number of solutions: " + str(len(self.solutions)) + "\n"
+        if self.tallies:
+            ret += "top solution tallies: " + str(self.tallies[0]) + "\n"
+        if self.energies:
+            ret += "top solution energy: " + str(self.energies[0]) + "\n"
+        if self.qubits:
+            ret += "qubits: " + str(self.qubits) + "\n"
+        if self.depth:
+            ret += "depth: " + str(self.depth) + "\n"
+        if self.times:
+            ret += "times: " + str(self.times) + "\n"
```

Comment:
It would be more user-friendly to format each time as a string so the user sees `2022-05-18 17:41:03.840047` instead of `datetime.datetime(2022, 5, 18, 17, 40, 59, 824668)`.